### PR TITLE
Cancellation Service Requested Changes #2

### PR DESCRIPTION
**Pull request by al-amine**

### CODE REVIEW

File: src/main/java/com/st/utopia/cancellationservice/model/Ticket.java
Comment (kingjon3377) [inline src/main/java/com/st/utopia/cancellationservice/model/Ticket.java]: By this line we *know* that `obj` is *not* a `Ticket`, so this will cause a `ClassCastException`. Instead, just return `false`. (I would put it in an `else` block, but that's a matter of stylistic preference.)

File: src/main/java/com/st/utopia/cancellationservice/model/TicketIdentity.java
Comment (kingjon3377) [inline src/main/java/com/st/utopia/cancellationservice/model/TicketIdentity.java]: Use braces around `if` and `else` blocks here as well (and in my opinion refactoring to the same pattern as you now use in `Ticket::equals` would be an improvement as well, but that's a less important issue).

File: src/main/java/com/st/utopia/cancellationservice/model/Ticket.java
Comment (kingjon3377) [inline src/main/java/com/st/utopia/cancellationservice/model/Ticket.java]: Getters and setters should have the same capitalization (except for capitalizing the letter after `get`, `set`, or `is`) as the field they access; in method names you consistently say `reservationtimeout` and `bookingid` when the fields are `reservationTimeout` and `bookingId`.

File: src/main/java/com/st/utopia/cancellationservice/service/TicketService.java
Comment (kingjon3377) [inline src/main/java/com/st/utopia/cancellationservice/service/TicketService.java]: This error message is not quite right: the method will return `null` when the ticket does not *exist*.

File: src/test/java/com/st/utopia/cancellationservice/dao/TicketDaoTest.java
Comment (kingjon3377) [inline src/test/java/com/st/utopia/cancellationservice/dao/TicketDaoTest.java]: Never use `assertTrue` or `assertFalse` to assert equality or inequality; use `assertEquals` or `assertNotEquals`.

**Review by kingjon3377 — CHANGES_REQUESTED** [review]
There are only a couple more stylistic things that I consider major enough to block merging of a PR (see comments inline). The lack of assertion messages in JUnit assertion-method calls is also troubling, but not worth blocking at this point.

It's much too late for this work, but in future PRs please follow the basic minimal guidelines for using version control generally and Git specifically:

- While the code should compile, and ideally run, without error at every commit, commit *early* and *often*: do *not* drop your entire "completed" work into version control as a single commit.
- Do *not* do your work (that will have to be merged via a PR) on the `master` branch; make a separate branch instead.

File: mvnw
Comment (kingjon3377) [inline mvnw]: Wasn't this removed already?

(I won't make its presence block merging, however.)

File: src/main/java/com/st/utopia/cancellationservice/model/TicketIdentity.java
Comment (kingjon3377) [inline src/main/java/com/st/utopia/cancellationservice/model/TicketIdentity.java]: Equality should depend on row and seat as well, *especially* since they are used in `hashCode()`.

**Review by kingjon3377 — CHANGES_REQUESTED** [review]
I'm still not happy with how your tests look like they either shouldn't pass at all or would pass even if the underlying code did nothing, but at this point (especially since I need your service in place to build my user-facing API against) the only change I'll say *needs* to be addressed before merging is to make `TicketIdentity::equals` reflect all of its fields, not just one.

File: src/main/java/com/st/utopia/cancellationservice/dao/TicketDao.java
Comment (STOzaki) [inline src/main/java/com/st/utopia/cancellationservice/dao/TicketDao.java]: Why do you have a logger if you never use it?

**Review by STOzaki — APPROVED** [review]
Looks good except for a few places, but because of the urgency. It is fine

**Review by kingjon3377 — APPROVED** [review]
Good enough for now.

**Pull request merged**
